libraries/AP_Camera/AP_Camera_Mount.cpp

```diff
@@ -78,4 +78,13 @@ void AP_Camera_Mount::send_camera_settings(mavlink_channel_t chan) const
     }
 }
 
+uint8_t AP_Camera_Mount::get_mount_instance_for_camera() const
```

Comment:
Let's put a little comment above here just to match other methods.  It's all pretty obvious what it does but still, I like comments! :-)